### PR TITLE
fix: let CsvVolumeLayer writer also write passive surfaces

### PR DESCRIPTION
**Pull request by asalzburger**
This PR includes a fix to the CsvTrackingGeometry writer that did omit passive layer volumes.

### CODE REVIEW

**Comment by asalzburger** [comment]
Hey @robertlangenberg  - as discussed yesterday, the changes look bigger in the file as they are.

 * the lambda function previously inlined has been refactored outside
 * the logic has been changed such that also passive layers are caught

File: Examples/Io/Csv/src/CsvTrackingGeometryWriter.cpp [hunk 1]
```diff
@@ -131,10 +218,10 @@ void writeVolume(SurfaceWriter& sfWriter, SurfaceGridWriter& sfGridWriter,
     std::vector<Acts::ActsScalar> volumeBoundValues =
         volume.volumeBounds().values();
     std::vector<Acts::ActsScalar> lastBoundValues;
+
     if (volume.volumeBounds().type() == Acts::VolumeBounds::eCylinder) {
       auto vTranslation = vTransform.translation();
-      // change volume Bound values to r min , r max , z min, z max, phi min ,
-      // phi max
+      // values to r min , r max , z min, z max, phi min, phi max
```
Comment (robertlangenberg) [inline Examples/Io/Csv/src/CsvTrackingGeometryWriter.cpp:224]: looks unintentional

File: Examples/Io/Csv/src/CsvTrackingGeometryWriter.cpp [hunk 2]
```diff
@@ -146,95 +233,29 @@ void writeVolume(SurfaceWriter& sfWriter, SurfaceGridWriter& sfGridWriter,
...
+      plvDims.max_v0 = volumeBoundValues[1];
+      plvDims.min_v1 = volumeBoundValues[2];
+      plvDims.max_v1 = volumeBoundValues[3];
+      lvWriter.append(plvDims);
+    }
+
+    // Now loop over the layer and write them
     for (auto layer : layers) {
-      // We jump navigation layers - they will be written last
+      // We jump navigation layers from layer volume writing
```
Comment (robertlangenberg) [inline Examples/Io/Csv/src/CsvTrackingGeometryWriter.cpp:256]:
```suggestion
      // We skip over navigation layers for layer volume writing
```

**Review by robertlangenberg — APPROVED** [review]
looks good to me

File: Examples/Io/Csv/src/CsvTrackingGeometryWriter.cpp (hunk 1 above)
Comment (asalzburger) [reply Examples/Io/Csv/src/CsvTrackingGeometryWriter.cpp:224]: That's ok, that's just a minor comment.

**Comment by asalzburger** [comment]
This has been approved.

**Pull request merged**